WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailFragment.java

```diff
@@ -102,9 +102,13 @@ public void onClick(View view) {
                 ActivityUtils.hideKeyboardForced(getActivity().getCurrentFocus());
 
                 if (NetworkUtils.checkConnection(getActivity())) {
-                    mOldSitesIDs = SiteUtils.getCurrentSiteIds(mSiteStore, false);
-                    isSocialLogin = true;
-                    addGoogleFragment();
+                    if (isAdded()) {
+                        mOldSitesIDs = SiteUtils.getCurrentSiteIds(mSiteStore, false);
+                        isSocialLogin = true;
+                        addGoogleFragment();
+                    } else {
+                        AppLog.e(T.NUX, "Google login could not be started.  LoginEmailFragment was not attached.");
```

Comment:
Is there something we could do other than fail silently in this situation? It seems odd to let the user tap "Log in with Google" and not have anything happen.

Reply:
I'm not a fan of failing silently either.  I'm not sure how/when the situation occurs to cause the crash to be honest.  The `onSaveInstanceState` call is occurring before the `FragmentTransaction.commit()` statement somehow, which could be something like the device is being rotated or the operating system is killing the activity, but I'm unable to reproduce the crash.  Since I'm not sure what's happening, I'm not sure how to inform the user other than a generic message that suggests contacting support.  Do you have something else in mind?

Reply:
I think a generic message is fine, just a note to let the user know something isn't working. I wouldn't suggest contacting support, though, since support wouldn't be able to help them. Something as simple as "Google login could not be started" would be fine, I think.

Reply:
I added the generic dialog message in 8d4e78802ec8d3a23cfac1a0580518052a582c4d.